src/sentry/integrations/vsts/integration.py

```diff
@@ -182,10 +187,10 @@ def get_organization_config(self):
                     for state in project_states:
                         all_states.add(state["name"])
 
-            all_states = [(state, state) for state in all_states]
+            all_states = {(state, state) for state in all_states}
             disabled = False
         except (ApiError, IdentityNotValid):
-            all_states = []
+            all_states = set()
```

Comment:
Why is this a set now?

Reply:
I was trying to be consistent to the original type of `all_states`:
```python
try:
    all_states = set()
    for idx, project in enumerate(projects):
		...
        all_states.add(state["name"])
    all_states = [(state, state) for state in all_states]
except (ApiError, IdentityNotValid):
    all_states = []
```
but you're right it should end up as an array.